source/content/relaunch.md

```diff
@@ -21,6 +21,20 @@ The relaunch process applies exclusively to live sites already hosted on Pantheo
 
 ## Prepare for Relaunch
 
+### Expected Disruption
+
+During this procedure, there will be a very brief period of disruption where requests to the custom domain will serve a `404 Unknown Site` error.
+
+The `404 Unknown Site` error will be returned as soon as the custom domain is removed from the old site, and once connected to the new site dashboard requests will immediately start to return a `200 OK` status once again. Requests will start routing to the new site dashboard after the TTL for the DNS records expires.
+
+There is no risk of HTTPS interruption during the relaunch procedure. Since launched domains already have a valid certificate, the existing certificate is served and valid until a new one is provisioned and deployed by the platform.  
```

Comment:
@rachelwhitton This sentence coupled with the previous sentences may be confusing to some readers. I guess what you are trying to inform is that while the users will see 404s, there is no need to reupload/attach certs during the relaunch procedure. Technically, "HTTPS" traffic will be interrupted since 404s basically is an interruption to the customer's traffic. Am I understanding this correctly ?

Reply:
We could remove this sentence, since certs are discussed in the first FAQ at the bottom of the doc